qutebrowser/keyinput/modeman.py

```diff
@@ -234,6 +236,7 @@ def enter(self, mode, reason=None, only_if_normal=False):
         if mode not in self._parsers:
             raise ValueError("No keyparser for mode {}".format(mode))
         prompt_modes = (usertypes.KeyMode.prompt, usertypes.KeyMode.yesno)
+        input_modes = [usertypes.KeyMode.insert, usertypes.KeyMode.passthrough]
```

Comment:
Given that we need this in a lot of places, we should probably have a `PROMPT_MODES` and `INPUT_MODES` constant in `modeman.py` and use that from other places as well.

Reply:
right, totally agree